client-appimage/Dockerfile

```
@@ -214,7 +214,7 @@ ENV PKG_CONFIG_PATH ${QT_ROOT}/lib/pkgconfig:${PKG_CONFIG_PATH}
 # Install QtKeychain
 RUN if [ "$BUILD_QT" != "1" ] ; then \
     cd /tmp && \
-    git clone https://github.com/frankosterfeld/qtkeychain.git --depth 1 -b v0.12.0 && \
+    git clone https://github.com/frankosterfeld/qtkeychain.git --depth 1 -b 0.14.2 && \
```

You removed the v here but not in the other file?

```suggestion
    git clone https://github.com/frankosterfeld/qtkeychain.git --depth 1 -b v0.14.2 && \
```

---

the lack of removal of `v` in the other one was the issue
I should have waited the few build hours before pushing
I just fixed it now